src/cr/cube/dimension.py

```diff
@@ -1233,6 +1239,30 @@ def _order_dict(self):
         return self._dimension_transforms_dict.get("order") or {}
 
 
+class _SmoothingSpec(object):
+    """Value object providing convenient access to details of a smoothing transform."""
+
+    def __init__(self, smoothing_dict):
+        self._smoothing_dict = smoothing_dict
+
+    @lazyproperty
+    def function(self):
+        """String representing the smoothing function specified in the payload.
+
+        Default smoothing function is ``one_sided_moving_avg``
+        """
+        return self._smoothing_dict.get("function", "one_sided_moving_avg")
```

Comment:
This will be more robust as:

```python
return self._smoothing_dict.get("function") or "one_sided_moving_avg"
```

This alternate expression handles the cases:

```python
"smoothing": {"function": null}
"smoothing": {"function": ""}
"smoothing": {"function": false or 0 or []}
```

Of all of these, the `null` case is most likely, but it's a welcome resiliency for any of the falsey cases.